webapp/src/components/link_preview/preview.css

```diff
@@ -0,0 +1,156 @@
+:root {
+    --light-gray: #6a737d;
+    --blue: #274466;
+    --light-blue: #eff7ff;
+    --github-merged: #6f42c1;
+    --github-closed: #cb2431;
+    --github-open: #28a745;
+    --github-not-planned: #6e7681
+}
```

Comment:
This css will affect the whole web application, which we don't want in this case. We can use sass variables here at compile time instead of setting these css vars

Reply:
Can I use scss on this project? I thought I couldn't use SCSS because I couldn't find the SCSS file on this repository.

Reply:
Yes `scss` is configured and should work in this project